portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -4467,6 +4467,87 @@ definition {
 			uploadFileName = "Document_1.jpg");
 	}
 
+	@priority = "4"
```

Comment:
@natocesarrego ,
I have created this based on Forms#SubmitFormWithUploadField. This way I've inherited the priority property. 
However, I've checked it with Cleyton and he shared with me that we could define the automated tests priority using the same [assessment matrix](https://docs.google.com/document/d/1bmvTGcj08Ihuw_CJbTP4TiFva_GCd0usgkNTcQTyenM/edit?ts=5edffe19#) that is used with the manual ones, so I believe it can remain as High.